packages/badge/src/css/index.js

```diff
@@ -3,8 +3,8 @@ import { transparentize } from 'polished'
 import core from '@pluralsight/ps-design-system-core'
 import { names as themeNames } from '@pluralsight/ps-design-system-theme/vars'
 
-import { buildCompoundClass } from '../js'
-import { appearances, colors } from '../vars'
+import { defaultWithColor, subtleThemeWithColor } from '../js'
```

Comment:
👍 this reads better.

obviously it would be nicer to just be able to add classes individually in the component and then use combos of css selectors to get all the variation. i don't yet have a path forward there...